src/main/java/com/sampleweb/pages/BabyPage.java

```diff
@@ -0,0 +1,56 @@
+package com.sampleweb.pages;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.stream.Collectors;
+
+public class BabyPage extends HomePage {
+
+    public static final String PATH="https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Dbaby&field-keywords=";
+
+    public String title = "Baby's Page";
+    public String pageTitle="//h1[text()=\"Baby Store\"]";
+    public String globalStoreXPath="//option[text()=\"Amazon Global Store\"]";
+    public String avgCustomerReviewXPath="//option[text()=\"Avg. Customer Review\"]";
+    public String subnav="//descendant-or-self::a[@tabindex=\"35\"]";
+    public String categoryXPath="//node()[text()=\"Baby\"]";
+    public String primeCheckBox="//input[@name=\"s-ref-checkbox-419158031\"]";
```

Comment:
this seems to be an autogenerated attribute name which could change in time. Consider a better attribute to identify element